msal/src/MSAL.Common.props

```diff
@@ -0,0 +1,204 @@
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net45'">
+    <DefineConstants>$(DefineConstants);DESKTOP</DefineConstants>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'Xamarin.iOS10'">
+    <DefineConstants>$(DefineConstants);iOS</DefineConstants>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
+    <DefineConstants>$(DefineConstants)</DefineConstants>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- This None is here so the conditionally included files show up in the Solution Explorer -->
+    <None Include="$(PathToMsalSources)\**\*.cs;$(PathToMsalSources)\**\*.xml;$(PathToMsalSources)\**\*.axml" 
+          Exclude="$(PathToMsalSources)\obj\**\*.*;$(PathToMsalSources)\bin\**\*.*" />
+    <!-- Manually include the cs files -->    
+    <Compile Include="$(PathToMsalSources)\**\*.cs" Exclude="$(PathToMsalSources)\obj\**\*.*" />    
+    <Compile Remove="$(PathToMsalSources)\Platforms\**\*.*" />
+    <Compile Remove="$(PathToMsalSources)\Features\**\*.*" />
+
+    <EmbeddedResource Include="$(PathToMsalSources)\Properties\Microsoft.Identity.Client.rd.xml" />
+    
+    <!-- Include Core as files, not as a project reference -->
+    <Compile Include="$(PathToCoreSources)\**\*.cs" Exclude="$(PathToCoreSources)\obj\**\*.*;$(PathToCoreSources)\properties\**\*.*" LinkBase="Core" />
+    
+    <None Include="$(PathToCoreSources)\Platforms\**\*.cs" LinkBase="Core\Platforms" />
+    <None Include="$(PathToCoreSources)\Features\**\*.cs" LinkBase="Core\Features" />
+    <None Include="$(PathToCoreSources)\*.md" LinkBase="Core" />
+    <Compile Remove="$(PathToCoreSources)\Platforms\**\*.cs" />
+    <Compile Remove="$(PathToCoreSources)\Features\**\*.*" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
+    <Compile Include="$(PathToMsalSources)\Platforms\netstandard1.3\**\*.cs" LinkBase="Platforms\netstandard1.3" />
+    <Compile Include="$(PathToCoreSources)\Platforms\netstandard1.3\**\*.cs" LinkBase="Core\Platforms\netstandard1.3" />
+    <Compile Include="$(PathToCoreSources)\Features\Cache\**\*.cs" LinkBase="Core\Features\Cache" />
+    <Compile Include="$(PathToMsalSources)\Features\PublicClientWithTokenCache\**\*.cs" LinkBase="Features\PublicClientWithTokenCache" />
+    <Compile Include="$(PathToMsalSources)\Features\ConfidentialClient\**\*.cs"  LinkBase="Features\ConfidentialClient" />
+    <Compile Include="$(PathToMsalSources)\Features\EventSource\**\*.cs" LinkBase="Features\EventSource" />
+    <Compile Include="$(PathToMsalSources)\Features\TokenCache\**\*.cs" LinkBase="Features\TokenCache" />
+    <Compile Include="$(PathToMsalSources)\Features\DeviceCode\**\*.cs" LinkBase="Features\DeviceCode"/>
+    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
```

Comment:
since these are now in a props file and not the csproj, will nuget update from within VS work properly?

Reply:
The VS experience looks unchanged on my machine. I think it is because it is using the new csproj flavour, where reloading is dynamic.